lib/active_support/cache/database_store.rb

```diff
@@ -20,8 +20,10 @@ def self.supports_cache_versioning?
 
       # param [Hash] options options
       # option options [Class] :model model class. Default: ActiveSupport::Cache::DatabaseStore::Model
+      # option options [Boolean] :auto_cleanup When true, runs {#cleanup} after every {#write_entry} and {#delete_entry}. Default: false
```

Comment:
Is there an example of any other store implementing this feature? I am not sure I agree with making this a boolean option and cleaning up on *every* write. I assume most users would want this to be a time-based option